contents/docs/data/persons.mdx

```diff
@@ -55,3 +55,63 @@ Clicking on a user opens their profile and shows all their properties.
 ## Deleting person data with the API
 
 You can also delete persons data via the API. See the [Data Deletion docs](/docs/privacy/data-deletion) for more information.
+
+## Events with person processing
+
+By default all events create or update an associated person profile (see above). This allows for the following features:
+
+- Person profiles
+- Filtering on person properties
+- Merging identities across distinct_ids (ie merging anonymous users with your logged in user)
+- Cohort creation
+- Feature flag targeting on properties
+- Tracking initial UTM tags and referrers over time
+
+However, there are many usecases that don't require this person processing. Because events that don't do person processing are cheaper for us to ingest, the pricing for these events is lower.
+
+Some examples of usecases that often don't require user profiles
+- Tracking a marketing website
+- B2C apps where users don't sign up or log in
+- High volume API server events
+
+### PostHog-js
+
+You can control whether events from PostHog-js have person processing enabled with the `process_persons` config setting. There are three options:
+
+- `process_persons: 'always'` _(default)_ - we will process persons data for all events
+- `process_persons: 'never'` - we won't process persons for any event. This means that anonymous users will not be merged once they sign up or login, so you lose the ability to create funnels that track users from anonyomous to identified. All events (including `$identify`) will be sent with `$process_persons: False`.
+- `process_persons: 'identified_only'` - we will only process persons when you call `posthog.identify([distinct_id])`. Anonymous users won't get person profiles.
```

Comment:
This isn't mentioned in the [ingestion RFC](https://github.com/PostHog/product-internal/pull/584/files#r1535795209). Note here to make sure these things line up.